src/doc/users-guide/install.md

```diff
@@ -99,42 +99,54 @@ the `--help` argument to the script.
 
 The OpenCilk C (or C++) compiler can be invoked via `bin/clang` (or
 `bin/clang++`) from within the installation directory.
+Alternatively, you can configure your system so that `clang` and `clang++` point to the OpenCilk C and C++ compilers, 
+as shown in the example below.
 
-{% alert "info" %}
-***Example:***
+### Example
```

Comment:
I think it looks better to put "Example" inside of the alert (as it was before), rather than outside.  That way the entire example is self-contained.

Reply:
Yes -- I tried it first with "Example" inside and the h2 formatting came out funny. If I can make a quick fix to the CSS, then I will put "Example" back inside.